src/Symfony/Component/Form/FormConfigBuilder.php

```diff
@@ -792,12 +794,8 @@ public static function validateName($name)
      *   * starts with a letter, digit or underscore
      *   * contains only letters, digits, numbers, underscores ("_"),
      *     hyphens ("-") and colons (":")
-     *
-     * @param string|null $name The tested form name
-     *
-     * @return bool Whether the name is valid
      */
-    public static function isValidName($name)
+    final public static function isValidName(?string $name): bool
```

Comment:
there is no point in people overwriting the method as its called with `self` instead of late static binding.